### PR TITLE
Issue #21398: Update exposeWebInfOnDispatch description.

### LINKED ISSUE #21398: Add additional details to `exposeWebInfOnDispatch` Server configuration description
**Describe the bug**  
The Server configuration: https://openliberty.io/docs/latest/reference/config/webContainer.html  description for `exposeWebInfOnDispatch` does not provide enough information.

**Steps to Reproduce**  
N/A

**Expected behavior**  
Add additional information to provide Open Liberty users with better information for the `exposeWebInfOnDispatch` configuration.

**Diagnostic information:**  
N/A

**Additional context**  
N/A

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #21398

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

File: dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -58,7 +58,7 @@ httpsIndicatorHeader.name=HTTPS indicator header
 httpsIndicatorHeader.desc=For SSL offloading, set to the name of the HTTP header variable inserted by the SSL accelerator/proxy/load balancer.
 
 exposeWebInfOnDispatch.name=Expose web info on dispatch
-exposeWebInfOnDispatch.desc=If true, a servlet can access files in the WEB-INF directory. If false (default), a servlet cannot access files the WEB-INF directory.
+exposeWebInfOnDispatch.desc=If true, a servlet can access static files in the WEB-INF directory. If false (default), a servlet cannot access static files the WEB-INF directory.
```
Comment (dmuelle) [inline dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties]:
```suggestion
exposeWebInfOnDispatch.desc=When this property is set to true, a servlet can access static files in the WEB-INF directory. When this property is set to false, which is the default, a servlet cannot access static files in the WEB-INF directory.
```

**Comment by pnicolucci** [comment]
@dmuelle looks ok to you now? Thanks for the suggestions!

**Comment by pnicolucci** [comment]
#build

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Review by pmd1nh — APPROVED** [review]

**Comment by pnicolucci** [comment]
#libby

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#libby

**Pull request merged**